apps/admin/backend/src/store.ts

```diff
@@ -951,24 +956,70 @@ export class Store {
   private deleteWriteInCandidateIfChildless(id: Id): void {
     const adjudicatedWriteIn = this.client.one(
       `
-      select id
-      from write_ins
-      where write_in_candidate_id = ?
-    `,
+        select id from write_ins
+        where write_in_candidate_id = ?
+      `,
+      id
+    ) as { id: Id } | undefined;
+
+    const manualTally = this.client.one(
+      `
+        select manual_tally_id from manual_tally_write_in_candidate_references
+        where write_in_candidate_id = ?
+      `,
       id
     ) as { id: Id } | undefined;
 
-    if (!adjudicatedWriteIn) {
+    if (!adjudicatedWriteIn && !manualTally) {
       this.client.run(
         `
-        delete from write_in_candidates
-        where id = ?
-      `,
+          delete from write_in_candidates
+          where id = ?
+        `,
         id
       );
     }
   }
 
+  private deleteAllChildlessWriteInCandidates(): void {
```

Comment:
could probably do all this in one query with nested select statements but I don't trust `sqlite3` to optimize the queries properly